src/main/webapp/app/guided-tour/guided-tour-step.model.ts

```diff
@@ -31,12 +13,58 @@ export interface TourStep {
     skipStep?: boolean;
```

Comment:
think about whether it makes sense to use the default value `false` here. This might simplify some other code, because you know that you always have either false or true and you don't really need to handle null or undefined here

Reply:
@krusche I have tried that out and the suggested change `skipStep = false` would define that every tour step now has to include the skipTour property in their object.

Reply:
that's weird, but let's leave it for now like this, we could still look into this in the future